util/arena/arena_test.go

```diff
@@ -16,73 +16,37 @@ package arena
 import (
 	"testing"
 
-	. "github.com/pingcap/check"
+	"github.com/stretchr/testify/assert"
 )
 
-func TestT(t *testing.T) {
-	TestingT(t)
-}
-
 func TestSimpleArenaAllocator(t *testing.T) {
 	arena := NewAllocator(1000)
 	slice := arena.Alloc(10)
-	if arena.off != 10 {
-		t.Error("off not match, expect 10 bug got", arena.off)
-	}
-
-	if len(slice) != 0 || cap(slice) != 10 {
-		t.Error("slice length or cap not match")
-	}
+	assert.Equal(t, 10, arena.off, "off not match, expect 10 bug got")
+	assert.False(t, len(slice) != 0 || cap(slice) != 10, "slice length or cap not match")
```

Comment:
We may use two positive assertions here.

```suggestion
	assert.Len(t, slice, 0)
	assert.Equal(t, 10, cap(slice))
```

Also the comment is redundant, `testify` will generate pretty error message.